internal/clusters_test.go

```diff
@@ -209,15 +209,21 @@ func TestAccClustersApiIntegration(t *testing.T) {
 	require.NoError(t, err)
 	assert.True(t, len(events) > 0)
 
+	// List clusters in workspace
 	all, err := w.Clusters.ListAll(ctx, compute.ListClustersRequest{})
 	require.NoError(t, err)
+	foundCluster := false
+	for _, info := range all {
+		if info.ClusterName == clusterName {
+			foundCluster = true
+		}
+	}
+	assert.True(t, foundCluster)
 
-	// List compute in workspace
-	names, err := w.Clusters.ClusterInfoClusterNameToClusterIdMap(ctx,
-		compute.ListClustersRequest{})
```

Comment:
Do you happen to know why did it start failing recently and not before?

Reply:
probably because i created a luster with the same name as another cluster...